flashcards/src/home.html

```diff
@@ -32,7 +33,7 @@ <h1 class="text text--xxl text--light text--shadow">Flashcards.com</h1>
         <h2 class="text text--lg">My Profile</h2>
         <div class="profile__content">
           <div class="profile__upload">
-            <img src="assets/images/user.png" alt="upload-image" />
+            <!-- <img src="" alt="" /> -->
```

Comment:
Any reason for comment this line

Reply:
@hungnguyenhuu-agilityio I forgot to remove it. I have updated it.